crates/holochain/src/core/workflow/call_zome_workflow.rs

```diff
@@ -100,8 +102,8 @@ where
                             }
                         }
                         None => {
-                            trigger_publish_dht_ops.trigger(&"trigger_publish_dht_ops");
-                            trigger_integrate_dht_ops.trigger(&"trigger_integrate_dht_ops");
+                            trigger_publish_dht_ops.trigger(&"call_zome_workflow");
```

Comment:
I don't know if these strings are used, but this just didn't make sense :) We know what's being triggered, we care who's triggering it or why